packages/inter-protocol/test/smartWallet/boot-psm.js

```diff
@@ -257,8 +259,10 @@ export const buildRootObject = async (vatPowers, vatParameters) => {
     };
 
     await Promise.all([
+      produceDiagnostics(allPowers),
       produceStartUpgradable(allPowers),
       produceStartGovernedUpgradable(allPowers),
+      produceHighPrioritySendersManager(allPowers),
```

Comment:
I wonder how I didn't need to add this earlier. I suppose you found a bug somewhere... fluxAggregator something something i suppose...

Reply:
this one is because some promise was waiting on it. some other changes were because it was being resolved with different values than the static types were saying. and there hadn't been a runtime check. I added that and iterated until tests and code aligned.